### ISSUE TITLE
Error when enabling Latex Module

### ISSUE THREAD

**Issue by mtreis86** [mfiano/doom-emacs-config#1]
When I turn on Latex with this config I get a Smartparans error:
`sp--update-pair-list: Pair ' was never defined, please specify closing delimiter in instead of passing ‘nil’`

**Comment by mfiano** [comment]
Yeah, it's a bug in Doom's smartparens settings. I'm not sure how to solve it.

**Comment by mtreis86** [comment]
Henrik wasn't able to replicate it. He said it is something to do with how SP changed the way sp-pair and sp-local-pair are handled. I disabled that whole codeblock in your .doom.d and it still error'd though. Figured at the least I would check and see if it is just my machine or not.
